docs/tutorial/optimization.rst

```diff
@@ -67,4 +67,9 @@ This trained model is now ready to be used for scoring future data points.
 >>> trained_model.score(2000, '2020-06-08')
 {'Success': True, 'IsLogTransformed': 1, 'LogTransformedAdjustedActual': 7.601402334583733, 'LogTransformedPrediction': 7.529710533463001, 'LogTransformedStdErr': 0.06217883425408564, 'LogTransformedCILower': 7.422390543346913, 'LogTransformedCIUpper': 7.62662106869458, 'AdjustedActual': 2000.000000000015, 'Prediction': 1861.566274906425, 'StdErr': 110.9167321105633, 'CILower': 1672.028177505716, 'CIUpper': 2051.104372307134, 'ConfLevel': 90.0, 'ExogenousHolidays': 0, 'IsAnomaly': False, 'IsAnomalyExtreme': False, 'AnomalyProbability': 0.7545715087682185, 'DownAnomalyProbability': 0.12271424561589073, 'UpAnomalyProbability': 0.8772857543841093, 'ModelFreshness': 0.1}
 
+.. Note :: Configuration optimization is an expensive process. This should be triggered when training runs for the first
+time on a new data. This optimal configuration can be stored to be used for training in future with newer data points.
+The optimization should be called again only when there is any change in the structural pattern of the data or the model
+starts under-performing.
```

Comment:
Have you tried running this locally? I think this may be a syntax error. Should be something like

```rst
.. note :: this is the note's title
    this is the note's body and they have to all be indented
```

Reply:
I have the other notes in the same format though

Reply:
Nope, yours is different. The other ones are all one super long line, no line breaks in the middle. If you want linebreaks you have to use the nested form, see this

![image](https://user-images.githubusercontent.com/12706268/91105600-54da6a00-e625-11ea-9f94-0def01af242f.png)

Reply:
I have removed the line breaks. This should now be aligned with the other notes we have